### PR TITLE
Make an npm package that can share data with JS clients

### PR DESCRIPTION
**Pull request by starsirius**
## Problem
Consuming this gem is straightforward for ruby clients like Gravity ([proof-of-concept PR](https://github.com/artsy/gravity/pull/14811)). For JS clients like Metaphysics or hybrid clients like Volt, it's not as easy.

## Solution
For JS clients, it might not be easy to share logic/code, but it might not be that bad to share data at least. This PR makes the gem also an npm package so that JS clients can install and use the shared data in build time ([proof-of-concept PR in Volt](https://github.com/artsy/volt/pull/5570) using this). The basic idea is to convert ruby data to a json file that can be consumed by JS clients.

### But we will still maintain 2 copies of data in this gem/package, right?
Yes, but with some automation, I hope the conversion can be hands-free and can make sure both are in-sync and version-controlled.

### CODE REVIEW

**Comment by erikdstock** [comment]
Is this something we can automatically build into github/gem releases or? Or is it a task that a dev needs to manually run? If the later maybe a git pre-commit hook to run the task would be good here.

**Comment by starsirius** [comment]
I assume the ruby gem might eventually have more logic than just the data that the npm package exposes, which means releasing a new version of the ruby gem might not always update the json of the npm package. I was thinking to set up a CI check on PRs to make sure the constants in the ruby gem and the json in the npm package are in-sync, and then we can release the gem and package independently, automated or not.

File: package/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = require("./data/data.json")
```
Comment (erikdstock) [inline package/index.js:1]: Maybe good to also export a VERSION

Comment (starsirius) [reply package/index.js:1]: Interesting. I was thinking versioning the package itself is sufficient for clients, but we can definitely add it if there is a need.

**Comment by starsirius** [comment]
OK I got some good feedback during Knowledge Share today. I'm going to merge this and give it a try in Volt. Still welcome any feedback, of course! 💪

**Pull request merged**